controllers/configurationpolicy_controller.go

```diff
@@ -1741,7 +1741,7 @@ func (r *ConfigurationPolicyReconciler) handleSingleObj(
 
 	if exists && !obj.shouldExist {
 		// it is a mustnothave but it exist, so it must be deleted
-		if strings.EqualFold(string(remediation), string(policyv1.Enforce)) {
+		if remediation.IsEnforce() {
```

Comment:
@JustinKuli is the plan to keep mustnothave to remove what the operator created but nothing beyond that or is the plant to still remove mustnothave until the whole uninstall logic is added?